connectors/microsoft-teams/element-templates/microsoft-teams-connector.json

```
@@ -692,6 +692,34 @@
         "equals": "sendMessageToChannel"
       }
     },
+    {
+      "label": "Content type",
+      "description": "The type of the content. Possible values are text and html.",
+      "group": "data",
+      "type": "Dropdown",
+      "value": "TEXT",
+      "choices": [
+        {
+          "name": "TEXT",
+          "value": "TEXT"
+        },
+        {
+          "name": "HTML",
+          "value": "HTML"
```

My suggestion would be here and everywhere else where users will read labels, to avoid capitalized labels or texts.

To me it looks strange. @bastiankoerber WDYT?

```suggestion
      "value": "TEXT",
      "choices": [
        {
          "name": "Text",
          "value": "TEXT"
        },
        {
          "name": "HTML",
          "value": "HTML"
```

---

i agree here